Arguments: 
  /Users/dinu/.nvm/versions/node/v14.15.4/bin/node /Users/dinu/.yarn/bin/yarn.js add --dev @parcel/next

PATH: 
  /Users/dinu/miniconda3/bin:/Users/dinu/miniconda3/condabin:/Users/dinu/.nvm/versions/node/v14.15.4/bin:/Users/dinu/.yarn/bin:/Users/dinu/.config/yarn/global/node_modules/.bin:/Users/dinu/.cargo/bin:/Users/dinu/work/GCP/google-cloud-sdk/bin:/anaconda3/bin:/Users/dinu/.cabal/bin:/Users/dinu/.ghcup/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/dinu/Applications/WebDriver:/Applications/Postgres.app/Contents/Versions/latest/bin:/opt/X11/bin:/Library/Apple/usr/bin:~/work/Flutter/flutter/bin

Yarn version: 
  1.22.5

Node version: 
  14.15.4

Platform: 
  darwin x64

Trace: 
  Error: https://registry.yarnpkg.com/@parcel%2fnext: Not found
      at Request.params.callback [as _callback] (/Users/dinu/.yarn/lib/cli.js:66988:18)
      at Request.self.callback (/Users/dinu/.yarn/lib/cli.js:140749:22)
      at Request.emit (events.js:315:20)
      at Request.<anonymous> (/Users/dinu/.yarn/lib/cli.js:141721:10)
      at Request.emit (events.js:315:20)
      at IncomingMessage.<anonymous> (/Users/dinu/.yarn/lib/cli.js:141643:12)
      at Object.onceWrapper (events.js:421:28)
      at IncomingMessage.emit (events.js:327:22)
      at endReadableNT (internal/streams/readable.js:1327:12)
      at processTicksAndRejections (internal/process/task_queues.js:80:21)

npm manifest: 
  {
    "name": "drawings_1.8",
    "version": "1.0.0",
    "main": "index.js",
    "license": "MIT"
  }

yarn manifest: 
  No manifest

Lockfile: 
  No lockfile
